workspaceitem-data-duplicates.md

```diff
@@ -0,0 +1,89 @@
+# WorkspaceItem data of identifiers sectionType
```

Comment:
I was surprised to found this endpoint as I expect a porting of the dspace-cris rest contract already in place
https://github.com/4Science/Rest7Contract/blob/main-cris/workspaceitem-data-detect-duplicate.md

Reply:
@abollini : While I understand your frustration that this feature is no longer a direct port from DSpace-CRIS, this new approach was clearly announced by @pnbecker back in January in all the prior PRs:
* https://github.com/DSpace/RestContract/pull/198#issuecomment-1899294323
* https://github.com/DSpace/DSpace/pull/8415#issuecomment-1898107717
* https://github.com/DSpace/dspace-angular/pull/1732#issuecomment-1898111164

I also remember this being discussed in a meeting in either Dec or January where @pnbecker noted they would need to refactor the approach if Library Code was still to take this work on.  Because this feature was deemed a priority for Steering, I said that sounded reasonable & no one raised objections in the meeting.

I was not aware of any objections myself to the new approach until you submitted your reviews.  The 4Science team has always been assigned as reviewers to these Duplicate Detection PRs as I wanted to get your feedback.

But at this point, I don't feel we can delay what is a "working feature" just because it doesn't align with DSpace-CRIS. This feedback about DSpace-CRIS alignment comes very far after our "Review/Test Deadline" (Feb 9), and it's now too late to address.

If, in the future, we decide the DSpace-CRIS implementation is a better approach...there is always the opportunity to migrate from one to the other (or similar) in DSpace 9.0 or later.  Just because one implementation is accepted, doesn't mean it cannot be replaced in a future release.